CONTRIBUTING.md

```diff
@@ -50,16 +50,17 @@ system already, but many of them are preinstalled on a Pivotal dev box.
 
 (Run all of these commands from the project root)
 
-ruby 2.1.2 
+ruby 2.1.2
 
-bundle
+bundler
```

Comment:
Is this meant to be the name of the software (`bundler`) or the name of the CLI command (`bundle`)?

I took it to be the name of the software.